readme.md

```diff
@@ -582,6 +582,7 @@
 - [RethinkDB](https://github.com/d3viant0ne/awesome-rethinkdb#readme)
 - [TinkerPop](https://github.com/mohataher/awesome-tinkerpop#readme) - Graph computing framework.
 - [PostgreSQL](https://github.com/dhamaniasad/awesome-postgres#readme) - Object-relational database.
+	- [Hacking](https://github.com/pghacking/awesome-postgres-hacking#readme) - Resources for PostgreSQL hacking.
```

Comment:
Looks like a whitespace here:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/6825b848-e467-4af5-9a6b-10da70b338ca" />

Please remove it, so it is the same as the other entries

Reply:
I see, thank you, makes sense